ArcGISRuntimeSDKQt_CppSamples/EditData/EditWithBranchVersioning/EditWithBranchVersioning.qml

```diff
@@ -0,0 +1,348 @@
+                Layout.margins: 5
+                validator: RegExpValidator { regExp: /\w{0,50}/ }
+            }
+
+            ComboBox {
+                id: accessComboBox
+                model: versionAccessModel
+                Layout.columnSpan: 2
+                Layout.fillWidth: true
+                Layout.alignment: Qt.AlignHCenter
+                Layout.margins: 5
+            }
+
+            TextField {
+                id: descriptionTextField
+                placeholderText: qsTr("Enter description")
+                Layout.columnSpan: 2
+                Layout.alignment: Qt.AlignHCenter
+                Layout.margins: 5
+            }
+
+            Row {
+                Layout.alignment: Qt.AlignRight
+                Layout.columnSpan: 2
+                Layout.margins: 5
+                height: childrenRect.height
+                spacing: 5
+
+                Button {
+                    Layout.margins: 5
+                    Layout.alignment: Qt.AlignRight
+                    text: qsTr("Create")
+                    onClicked: {
+                        model.createVersion(versionNameTextField.text, accessComboBox.currentValue, descriptionTextField.text);
+                        resetCreateVersionWindow();
+                    }
+                }
+
+                Button {
+                    Layout.alignment: Qt.AlignRight
+                    Layout.margins: 5
+                    text: qsTr("Cancel")
+                    onClicked: {
+                        resetCreateVersionWindow();
+                    }
+                }
+            }
+        }
+    }
+
+    // Update Window
+    Rectangle {
+        id: updateWindow
+        anchors.centerIn: parent
+        width: childrenRect.width
+        height: childrenRect.height
+        radius: 10
+        visible: false
+
+        GaussianBlur {
+            anchors.fill: updateWindow
+            source: view
+            radius: 40
+            samples: 20
+        }
+
+        MouseArea {
+            anchors.fill: parent
+            onClicked: mouse.accepted = true;
+            onWheel: wheel.accepted = true;
+        }
+
+        GridLayout {
+            columns: 2
+            anchors.margins: 5
+
+            Text {
+                Layout.columnSpan: 2
+                Layout.margins: 5
+                Layout.alignment: Qt.AlignHCenter
+                text: qsTr("Update Attributes")
+                font.pixelSize: 16
+            }
+
+            Text {
+                text: "TYPDAMAGE:"
+                Layout.margins: 5
+            }
+
+            ComboBox {
+                id: typeDmgCombo
+                property int modelWidth: 0
+                Layout.minimumWidth: modelWidth + leftPadding + rightPadding + indicator.width
+                Layout.margins: 5
+                Layout.fillWidth: true
+                enabled: !model.sgdbVersionIsDefault
+                model: featAttributes
+            }
+
+            Row {
```

Comment:
Same question on Rows

Reply:
@anmacdonald
I would prefer to leave this one as is since not all of the children span both columns, albeit most of them did, and I was using this for formatting the UI of the buttons to be closer where they would be awkwardly spaced if in two columns.

with row:
![image](https://user-images.githubusercontent.com/11935261/98035776-e0af0600-1dcd-11eb-9cff-a40b15bd75d9.png)

no row:
![image](https://user-images.githubusercontent.com/11935261/98035802-eb699b00-1dcd-11eb-860e-86aa3350a973.png)